BackEndApp/authv1/views.py

```diff
@@ -93,3 +94,65 @@ def logout(request):
         status = 500
 
     return JsonResponse({"message": message}, status=status)
+
+
+@api_view(["POST"])
+def ForgotPassword(request):
+    try:
+        username = request.data.get("username")
+        user = User(username=username, password=None)
+        user = user.find()
+
+        if user is None:
+            status = 401
+            message = "User does not exist"
+
+        email = user["email"]
+        subject = "Auto-DL Request for Reset password"
+        msg = (
+            "You ("
+            + username
+            + ") requested a password reset. Click here to reset your password http://127.0.0.1:8000/auth/password-reset/"
```

Comment:
We shouldn't use 127.0.0.1 directly, would need to change in case of deployment, please change accordingly 😅

Reply:
I need a little help with this...